apps/pie-docs/src/_11ty/shortcodes/tokensTable/tokenTypes/colour.js

```diff
@@ -1,19 +1,5 @@
 const { isColorDark } = require('../../../../_utilities/colors');
 
-const createHighContrastName = (tokenName) => {
```

Comment:
These have been moved to `./utilities/tokens` so they can be shared between `tokenTable` and `simpleTable`